### PR TITLE
fix(outputs): fill compliance field for outputs

### PR DESCRIPTION
**Pull request by pedrooot**
### Description

The field compliance from outputs was empty for Azure and Kubernetes provider. This pr fixes this issue.

Thanks Oleksandr Lopatin for the report via Slack 🚀 

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/lib/outputs/compliance/compliance.py [hunk 1]
```diff
@@ -212,7 +212,7 @@ def get_check_compliance(finding, provider_type, output_options) -> dict:
                 compliance_fw = compliance.Framework
                 if compliance.Version:
                     compliance_fw = f"{compliance_fw}-{compliance.Version}"
-                if compliance.Provider == provider_type.upper():
+                if compliance.Provider.upper() == provider_type.upper():
```
Comment (jfagoagas) [inline prowler/lib/outputs/compliance/compliance.py:217]: 1. Please add tests for this function for other providers than AWS :D

Comment (jfagoagas) [inline prowler/lib/outputs/compliance/compliance.py:217]:
```suggestion
                # compliance.Provider == "Azure"
                # provider_type == "azure"
                if compliance.Provider.upper() == provider_type.upper():
```

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/lib/outputs/compliance/compliance.py (hunk 1 above)
Comment (pedrooot) [reply prowler/lib/outputs/compliance/compliance.py:217]: Done

Comment (pedrooot) [reply prowler/lib/outputs/compliance/compliance.py:217]: Done

**Review by jfagoagas — APPROVED** [review]
🔝

**Pull request merged**